addons/website_sale_stock/tests/test_website_sale_stock_product_warehouse.py

```diff
@@ -62,31 +62,37 @@ def test_01_get_combination_info(self):
         to the warehouse which is set in current website.
           - Set Warehouse 1, Warehouse 2 or none in website and:
             - Check available quantity of Product A and Product B in website
-        When the user doesn't set any warehouse, the module should still select
-        a default one.
+        When the user doesn't set any warehouse, the module should take into account availability in
+        all warehouses.
         """
         test_env = self.env['base'].with_context(
             website_id=self.website.id,
             website_sale_stock_get_quantity=True,
         ).env
 
-        for wh, qty_a, qty_b in [(self.warehouse_1, 10, 0), (self.warehouse_2, 15, 10), (False, 10, 0)]:
+        for wh, qty_a, qty_b in [
+            (self.warehouse_1, 10, 0),
+            (self.warehouse_2, 15, 10),
+            (False, 25, 10)
+        ]:
             # set warehouse_id
             self.website.warehouse_id = wh
 
             combination_info = self.product_A.with_env(test_env)._get_combination_info_variant()
 
             # Check available quantity of product is according to warehouse
+            formatted_wh = f'warehouse {wh}.' if wh else 'in all warehouses.'
             self.assertEqual(
                 combination_info['free_qty'], qty_a,
-                f"{qty_a} units of Product A should be available in warehouse {wh}")
+                f'{qty_a} units of Product A should be available in {formatted_wh}'
```

Comment:
I'm not a big fan of those messages. IMO, it's better to formulate the assertion in a way that makes the message redundant. And if you split your test, I think it will be redundant :)